logs/tickets/README.md

```diff
@@ -0,0 +1 @@
+Logs of ticket stats are stored in this directory.
```

Comment:
Are we still using flat files? How do we feel about SQLite here?

I suppose `JSON.stringify` is much easier than writing an ORM for SQLite. But it looks like you're doing CSV. At which point, we might want to seriously consider SQLite.

Reply:
(If SQLite is too much work, I would recommend at least TSV over CSV; it has less escaping concerns because you can just sanitize tabs out.)

Reply:
I had recommended TSV as well. I think its the way to go for this.

Reply:
re: TSV, I feel like I'm missing something on the use of TSV, It can't be as simple as just seperating values with tabs rather than spaces can it? None of the values used here should have commas in them so it seems wasteful to just do this. Correct me if i'm wrong about how a TSV file is setup/its use here vs CSV.

re: SQLite, have never done that but would be willing to look into it potentionally.

Reply:
> re: TSV, I feel like I'm missing something on the use of TSV, It can't be as simple as just seperating values with tabs rather than spaces can it?

It is exactly as simple as separating values with tabs rather than _commas_. 

> None of the values used here should have commas in them so it seems wasteful to just do this.

Yeah, because you changed what would more logically be commas into `;` because youre using CSV. Zarel and I are saying: please use `\t` and `,` instead of `,` and `;`.